<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Alephs-MacBook-Pro.local" failures="2" tests="3" name="invalidGet" time="0.868" errors="0" timestamp="2023-03-19T13:48:55 ICT">
  <testcase classname="Lab8.Example2Test" name="TS01_StatusCodeChecked" time="0.001"/>
  <testcase classname="Lab8.Example2Test" name="TS02_MessageChecked" time="0.002">
    <failure type="java.lang.AssertionError" message="Message check failed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Message check failed expected [true] but found [false]
at Lab8.Example2Test.TS02_MessageChecked(Example2Test.java:22)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- TS02_MessageChecked -->
  <testcase classname="Lab8.Example2Test" name="TS03_MessageContentChecked" time="0.011">
    <failure type="java.lang.AssertionError" message="Message content check failed expected [User not found!] but found [null]">
      <![CDATA[java.lang.AssertionError: Message content check failed expected [User not found!] but found [null]
at Lab8.Example2Test.TS03_MessageContentChecked(Example2Test.java:26)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- TS03_MessageContentChecked -->
</testsuite> <!-- invalidGet -->
